README.md

```diff
@@ -55,7 +55,7 @@ All of the Node.js Foundation websites, including this repo, are jointly governe
 - Rod Vagg ([rvagg](https://github.com/rvagg)): [@rvagg](https://twitter.com/rvagg), `rod``@``vagg.org`
 - Trent Oswald ([therebelrobot](https://github.com/therebelrobot)): [@therebelrobot](https://twitter.com/therebelrobot), `trentoswald``@``therebelrobot.com`
 
-### Website Working Group Members
+### Website Working Group Collaborators
```

Comment:
@phillipj It had the same wording until I foolishly changed it in my last commit tot hat file. ;)